systemd/systemd.go

```diff
@@ -429,7 +465,7 @@ func NewEmulationMode(rootDir string) Systemd {
 	if rootDir == "" {
 		rootDir = dirs.GlobalRootDir
 	}
-	return newSystemd(EmulationModeBackend, rootDir, SystemMode, nil)
+	return newSystemd(EmulationModeBackend, rootDir, SystemMode, nil, legacySystemd())
```

Comment:
This is for preseeding which operates on a given root anyway, so I guess the 'legacy' check does not matter there much.